packages/cli/src/commands/transfer/celo.ts

```diff
@@ -29,19 +28,19 @@ export default class TransferCelo extends BaseCommand {
     const value = new BigNumber(res.flags.value)
 
     this.kit.defaultAccount = from
-    const celoToken = await this.kit.contracts.getGoldToken()
+    const usdToken = await this.kit.contracts.getStableToken()
```

Comment:
Wooow. Nice catch!
Was a change added in a merge, without any conflict.... (1e649a88b3b9c5c07df6bf2b9190c58bcb1c1477)

Ok, this does not reassure me one bit.......

I'm changing this as it is in master. And I will review all the differences again just in case something like this happened with other file.

Again, amazing catch

Reply:
Ok, double checked. It was my mistake and it was in this commit (cf27409a1350a3efb52a491bf4ee5c15a2485d51)

So, my bad. I don't know what I was thinking...
At least my faith in git merging was restored, haha